packages/bpk-foundations-android/tokens/base.raw.android.json

```diff
@@ -1196,7 +1223,7 @@
       "name": "TEXT_XXL_FONT_WEIGHT"
     },
     "TEXT_XXXL_FONT_SIZE": {
```

Comment:
to be able to use the new sizes we'll also have to create these text aliases (i believe this is similar on iOS) - i think this is mainly where the confusion around the text style naming came from. i suspect we _could_ define this as `TEXT_HERO_1_FONT_SIZE` or similar to match the new naming for the new styles